OneCommunityGlobal/HighestGoodNetworkApp
# Anny fix reports view of team name

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98345990/0c94159f-fb66-40c8-b9d1-8338bad095b0)


## Main changes explained:
-Render only 5 teams detail once to speed up the loading time

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to Other Links → Reports → Reports → Teams
5. open any team on the right in a new tab, check if it render properly
6. select another team name from the list, test if it displays the new selected team

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98345990/c0e40d21-cdb7-4930-9c82-edf75c131f5c


## Note:
Although the loading time is faster than before, but it still take some time to load.
